src/Components/Register/Register.js

```diff
@@ -21,59 +21,155 @@ const Register = ({ onSubmit, errorLabel }) => {
     password: '',
     repeatPassword: '',
     alterEgo: '',
-    icon: '' });
+    icon: '',
+    isPwdMatched: false
+  });
 
   const handleChange = ({ target }) => {
-    const { name, value } = target;
+
+  const { name, value } = target;
+
+  let checkPwdValidity = {};
+    if ( name === 'password' ) {
```

Comment:
How about using a switch instead of ifs?

Reply:
Done.